capabilities_matrix/_topics/cloud_providers.md

```diff
@@ -2,60 +2,60 @@
+| Volume Creation/Deletion                   | No                        | No              | No                          | Yes                        | Yes                                   | Yes                               |
+| Instance Migration                         | No                        | No              | No                          | Yes                        | Yes                                   | No                                |
+| Automation Work Flows                      | Yes                       | No              | Yes                         | Yes                        | Yes                                   | No                                |
+| Network Manager - Read Only                | No                        | Yes             | Yes                         | No                         | Yes                                   | Yes                               |
+| Network Manager - CRUD Actions             | No                        | No              | No                          | No                         | No                                    | No                                |
+| Storage Manager                            | Yes                       | No              | No                          | Yes                        | No                                    | Yes                               |
+| Provision from Image to Instance           | Yes                       | Yes             | Yes                         | Yes                        | Yes                                   | Yes                               |
+| Provision from Image Snapshot to Instance  | No                        | No              | No                          | Yes                        | No                                    | No                                |
+| Provision from Volume to Instance          | No                        | No              | No                          | Yes                        | No                                    | No                                |
+| Provision from Volume Snapshot to Instance | No                        | No              | No                          | Yes                        | No                                    | No                                |
+| Orchestration Template Provisioning        | Yes                       | Yes             | No                          | Yes                        | No                                    | No                                |
+| Cloud-Init                                 | No                        | No              | No                          | Yes                        | No                                    | Yes                               |
+| Sysprep Windows Templates                  | No                        | No              | No                          | Yes                        | No                                    | No                                |
```

Comment:
Should we say N/A as this isn't supported by PowerVS - no somewhat gives the implication that it's a gap?

Reply:
(Sysprep Windows)

Reply:
done